openquake/commonlib/logictree.py

```diff
@@ -1108,7 +1108,7 @@ def collect_source_model_data(self, branch_id, source_model):
         self.source_ids[branch_id].extend(ID_REGEX.findall(xml))
         self.source_types.update(SOURCE_TYPE_REGEX.findall(xml))
 
-    def apply_uncertainties(self, ltpath, sm):
+    def apply_uncertainties(self, ltpath, sm, converter):
```

Comment:
pls update the description of parameters. What's a 'converter'?

Reply:
Done